assets/css/main.css

```diff
@@ -1,624 +1,693 @@
+/*
+Red/Teal
+rgba(204, 40, 41, 1), #4ba58c
+*/
+/*
+Grey-Blue
+
+*/
 html, body {
   margin: 0;
   padding: 0;
   height: 100%;
-  font-family: helvetica, sans-serif;
+  font-family: 	"mr-eaves-xl-modern", helvetica;
+  font-weight:400;
   line-height: 1.3;
-  color: #2b2b2b;
+  color:rgba(37,52,62, 1);
   background: #FBFCFC;
   font-size: 110%;
 }
 
-a {
-  color: #F1853D;
-  text-decoration: none;
-}
-
-button {
-  background: #F1853D;
-  border: 0;
-  color: #fff;
-  padding: 13px 30px;
-  font-size: 1.35em;
-}
-
-input,textarea {
-  font-size: inherit;
-  font-family: inherit;
-  -webkit-appearance: none;
-  border-radius: 0;
-}
-
-fieldset {
-  border: 0;
-  padding: 0;
-  margin: 0;
-}
-
-fieldset[disabled], button[disabled] {
-  opacity: 0.5;
-  filter: alpha(opacity=50);
+div{
+  display:inline-block;
+  width:100%;
 }
 
 nav ul {
   list-style: none;
 }
 
-@media screen and (max-width: 650px) {
-  body {
-    font-size: 95%;
-  }
-}
+body {
+  font-size: 95%;
+}
+
+h1, h2, h3, h4, h5, h6{
+  font-family: 	"mr-eaves-xl-modern";
+  font-weight:700;
```

Comment:
since family and weight are set here they can be removed from the individual header tag rules